transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -146,17 +143,13 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
     descriptor: Msg
   ): Msg = {
     val (entityId, entityVersion) = getEntityIdAndVersion(fileName)
-    val dataFileName = descriptor.read[String]("file_name")
     // put values in the form we want
     Obj(
-      mutable.LinkedHashMap[Msg, Msg](
-        Str(s"${entityType}_id") -> Str(entityId),
-        Str("version") -> Str(entityVersion),
-        Str("content") -> Str(encode(metadata)),
-        Str("virtual_path") -> Str(s"/$dataFileName"),
```

Comment:
did you intend to drop the virtual_path?

Reply:
Yes. Instead of using the file metadata to generate the bulk request, we use the raw descriptor. So there's no need for these extra top-level fields.